packages/RPM/SPECS/fledge.spec

```diff
@@ -462,8 +462,14 @@ copy_fledge_sudoer_file
 echo "Enabling Fledge service"
 enable_fledge_service
 
-echo "Starting Fledge service"
-start_fledge_service
+# Skipping the start of fledge when package is upgraded, 
+# because of other related packages upgrade transaction mechanism is causing issues in Fledge (FOGL-5506).
+if [ $1 == 2 ];then
+    echo "Please start fledge once upgrade process is complete; using 'systemctl start fledge'"
```

Comment:
We can avoid this manual intervention. It can be added in %postun script!


https://github.com/fledge-iot/fledge-pkg/blob/develop/packages/RPM/SPECS/fledge.spec#L516
```
systemctl start fledge
```